### PR TITLE
[client] vote-button / create-new-ask : Fix duplicate vote&creation issue

### PR DESCRIPTION
**Pull request by luckypapa**
Prevent duplicate vote when user quickly click vote button

### CODE REVIEW

**Comment by JungoKim** [comment]
+1

**Comment by luckypapa** [comment]
update additional patch

**Comment by luckypapa** [comment]
Rollback [client] select-target : Fix wrong prop value about DropDownMenu patch
-> this problem is upstream bug... so if we update material-ui verion, warning will be fixed

File: client/src/app/components/create-new-ask.jsx [hunk 1]
```diff
@@ -241,6 +249,7 @@ var CreateNewAsk = React.createClass({
       error: function (xhr, status, err) {
         this.setState({result: "Ask create fail.."});
         this.refs.snackbar.show();
+	this.setState({sent: false});
```
Comment (JungoKim) [inline client/src/app/components/create-new-ask.jsx]: make a proper indentation at next time

**Comment by JungoKim** [comment]
+1

**Pull request merged**